client/js/navigation/NavigationLayout.js

```diff
@@ -10,20 +14,28 @@ import ItemsPageScreen from '../screens/ItemsPage';
 import ItemScreen from '../screens/Item';
 import StoreScreen from '../screens/Store';
 import BrandScreen from '../screens/Brand';
+import {BackButton} from './config';
+
 const BrowseStack = createStackNavigator(
   {
     browse: {
       screen: BrowseScreen,
     },
     items: {
       screen: ItemsPageScreen,
+      navigationOptions: navigation => ({
+        headerLeft: <BackButton navigation={navigation} />,
+      }),
     },
     item: {
       screen: ItemScreen,
+      navigationOptions: () => ({
+        headerLeft: <BackButton />,
```

Comment:
Please pass on the navigation prop here as you did in items else you won't be able to access it inside the component.